docs/final report/chapters/feature-description.tex

```diff
@@ -1,11 +1,57 @@
 \section{Analysis}
+To analyse the data, the user has to switch to the 'analysis' tab. In this tab a text area is present where the user can add code in the program exclusive "SUGAR" language. When a script already exists, this file can simply be loaded into the program. This script can be changed and executed to affect the data. After changes are made, the script can be saved to file again. 
+
 \subsection{The C's}
-\subsection{Additional Analysis}
+Multiple different analyses exist to execute on data. Following the terminology of EDA we implemented 6 out of the Eight C's of the generic ESDA process:
+%TODO reference to blackboard document "Exploratory Sequential Data Analysis: Foundations" by C. Fisher of the MAYA Design Group
+\subsubsection{Constraint}
+A constraint can be used to select specific values from a column. Numbers can be compared to constants or other columns. Constraints can also be combined with AND, OR and NOT operations. On date values, the extra constraints "before" and "after" are available. 
+\subsubsection{Computation}
+Number values can be added, divided, multiplied and subtracted. The modulo operation is available, as well ass the square root and the power function. Date values can be added or subtracted and the relative difference can be computed. Separate date and time can be combined to one DateTime value, also the opposite; DateTime values can be split up to either date or time value. All the computations can also be used in the constraints in combination with column values or constants.
+\subsubsection{Comparison}
+To determine a relation between events, the lag sequential comparison is available. The lag sequential analysis will return a table chronologically sorted on the events.
+%lag sequential
+\subsubsection{Code}
+Codes can be added to rows when a value of that row satisfies a constraint. A single row can contain zero or more codes. When a row contains a code, it can be selected via a specific constraint that selects only that code. 
+\subsubsection{Chunking}
+Real chunks are not supported by the program, however the group by function is. A group by function can be applied on a column or on a constraint combined with a column. This creates a new table with the first column the value of the group, the following columns are created after the functions given by the user, applied on that column.
```

Comment:
we voldoen aan wat is gespecificeerd in de requirements. dus ik vind het een beetje raar om dan te stellen dat "Real chunks are not supported"